pkg/northbound/gnmi/set_test.go

```diff
@@ -110,6 +110,86 @@ func Test_doSingleSet(t *testing.T) {
 	assert.Equal(t, (*change.TypedUint64)(&newChange.Config[0].TypedValue).Uint(), uint(16))
 }
 
+// Test_doSingleSet shows how a value of 1 list can be set on a target
+func Test_doSingleSetList(t *testing.T) {
+	server, _ := setUp()
+	var deletePaths = make([]*gnmi.Path, 0)
+	var replacedPaths = make([]*gnmi.Update, 0)
+	var updatedPaths = make([]*gnmi.Update, 0)
+	pathElemsRefs, _ := utils.ParseGNMIElements(utils.SplitPath("/cont1a/cont2a[lista/b=c]/leafList"))
```

Comment:
there's never going to be a slash in the key name like "lista/b" it can only ever be in the key value
It should be /cont1a/list2a[name=a/b]/tx-power to align with the test1@2018-02-20.yang